src/main/java/com/google/devtools/build/lib/bazel/rules/android/android_ndk_platform_mappings.txt

```diff
@@ -0,0 +1,13 @@
+platforms:
+  @androidsdk//:android_arm
+    --android_cpu=armeabi-v7a
+    --fat_apk_cpu=armeabi-v7a
+  @androidsdk//:android_x86_64
+    --android_cpu=x86_64
+    --fat_apk_cpu=x86_64
+
+flags:
+  --fat_apk_cpu=armeabi-v7a
```

Comment:
See comment in main thread, I don't think this will work. The reason is that you want to apply the mappings _after_ the Android split transition occurred. Instead these mappings should operate on the `--cpu` values used for Android. Also for section above and below.